sdk/appconfiguration/Azure.Data.AppConfiguration/perf/Azure.Data.AppConfiguration.Perf.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\..\..\common\Perf\Azure.Test.Perf\Azure.Test.Perf.csproj" />
```

Comment:
Better to add 'Azure.Test.Perf.csproj' to the solution as well. That ensures it is a stand-alone solution.